src/main/java/com/impactupgrade/nucleus/controller/TwilioController.java

```diff
@@ -175,6 +178,43 @@ public Response inboundSignup(
     return Response.ok().entity(response.toXml()).build();
   }
 
+  /**
+   * This webhook serves as a more generic catch-all endpoint for inbound messages from Twilio.
+   */
+  @Path("/inbound/sms/catchall")
+  @POST
+  @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+  @Produces(MediaType.APPLICATION_XML)
+  public Response inboundKeyword(
+      Form rawFormData,
+      @Context HttpServletRequest request
+  ) throws Exception {
+    MultivaluedMap<String, String> smsData = rawFormData.asMap();
+
+    Environment env = envFactory.init(request);
+
+    List<String> optInKeywords = Arrays.asList("START", "UNSTOP", "YES");
```

Comment:
As far as I could tell from reading in Twilio's docs, these are all the default start/stop keywords that Twilio uses to opt in/out. Could have missed some, but this is a safe start either way.